shaders/program/shadowcomp1.glsl

```diff
@@ -410,8 +415,19 @@ void main() {
 
         if (insideFrustrum) {
             float thisDFval = getDistanceField(vxPos);
-            if (thisDFval < 0.7) {
-                if (thisDFval > 0.1) {
+            if (thisDFval < 0.7 || nextUint() % 37 == 0) {
+                bool isOccluded = (imageLoad(occupancyVolume, coords).r & 1) != 0;
+                float maxDFVal = thisDFval;
+                for (int k = 0; k < 3; k++) {
+                    float dplus = getDistanceField(vxPos + mat3(0.5)[k]);
+                    float dminus = getDistanceField(vxPos - mat3(0.5)[k]);
+                    if (isOccluded) dplus = -dplus;
```

Comment:
could do a single if check, would be closer to my code style but probably doesn't matter much